docs/source/faq/faq.rst

```diff
@@ -206,34 +206,31 @@ Where can I get more information about VISA?
 
 * The original VISA docs:
 
-  - `VISA specification`_ (scroll down to the end)
-  - `VISA library specification`_
-  - `VISA specification for textual languages`_
-
-* The very good VISA manuals from `National Instruments's VISA`_:
-
-  - `NI-VISA User Manual`_
-  - `NI-VISA Programmer Reference Manual`_
-  - `NI-VISA help file`_ in HTML
-
-.. _`VISA specification`:
-       http://www.ivifoundation.org/Downloads/Specifications.htm
-.. _`VISA library specification`:
-       http://www.ivifoundation.org/Downloads/Class%20Specifications/vpp43.doc
-.. _`VISA specification for textual languages`:
-       http://www.ivifoundation.org/Downloads/Class%20Specifications/vpp432.doc
-.. _`National Instruments's VISA`: http://ni.com/visa/
-.. _`NI-VISA Programmer Reference Manual`:
-       http://digital.ni.com/manuals.nsf/websearch/87E52268CF9ACCEE86256D0F006E860D
-.. _`NI-VISA help file`:
-       http://digital.ni.com/manuals.nsf/websearch/21992F3750B967ED86256F47007B00B3
-.. _`NI-VISA User Manual`:
-       http://digital.ni.com/manuals.nsf/websearch/266526277DFF74F786256ADC0065C50C
+  - `VISA/IVI/SCPI specifications`_ (scroll down to find VISA)
+
+* The very good VISA manuals from NI:
 
+  - `NI-VISA Download`_
+  - `NI-VISA help file`_ in HTML (download in PDF, upper right corner)
+  - `R&S-VISA Download`_
+  - `R&S Basics of Instrument Remote Control`_
+  - `Keysight-VISA Download`_
```

Comment:
Since we also have a page dedicated to getting NI-VISA, it would make sense to move the download links there and maybe rework this a bit to be less NI centered.

Reply:
I absolutely agree this should be less NI-VISA centric.  Let me give it a shot.